### DISCUSSION TITLE
Personal-Websites Index

### DISCUSSION THREAD

**Discussion by erlend-sh** [muni-town/weird#220] [Ideas]
As a moonshot idea, I hope we will one day have created a multi-million-entry index of ‘People’s Personal (but public) Websites’.

There are many precedents to this kind of thing:

https://indiemap.org/
https://github.com/snarfed/indie-map

https://blog.kagi.com/small-web
https://github.com/kagisearch/smallweb

https://ooh.directory/
https://ooh.directory/blog/2024/blog-pages/

https://personalsit.es/

What’s a little shocking is that we don’t yet have a good map of the Personal Sites of the world wide web. In grand total, how many people have established an independent web persona?

Meta knows exactly how many people are on Instagram, and we have a pretty good idea of it as well.

Personal sites though? Maybe there’s a million of them, maybe 20? (wouldn’t that be nice). We truly don’t know.

I think there’s power in knowing our own network size, so we can see ourselves on the map and fully appreciate our network-power.

![image](https://github.com/user-attachments/assets/e85d0dd6-07b2-49d4-b74f-5f0f89542a40)
https://www.deviantart.com/jaysimons/art/Map-of-Social-Media-Forums-and-Imageboards-877912094

https://www.visualcapitalist.com/the-online-world-mapped-2021/

Related:
- https://github.com/muni-town/weird/discussions/155

**Comment by erlend-sh** [comment]
Another way to put it is that we need to make a *protocol for webrings*. Personal website owners need a web-native way of opting in to various webring directories.

This kind of thing is cool: https://webring.xxiivv.com/
..but we can’t be relying on pull requests to make these connections. It has to be done natively through through the websites themselves, probably with many alternatives: rel-links, webfinger, atproto, activitypub, leaf etc. (the more static-friendly the better).

**Reply by zicklag** [reply]
Interesting, thought. So if you wanted to build a webring _index_ of sorts, you could maybe make up a `rel` flag for links, like `rel="webring"` to say that you want to make the site you're linking to as a "friend" and then if their site links back to your site with `rel="webring"`, then a search index and automatically establish the mutual webring connection between you.

Then you just need a service that will create the webring page.

There's lots to think about here, but I think there's some ways forward.
